circle.yml

```diff
@@ -0,0 +1,22 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:4
+    steps:
+      - checkout
+      - restore_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
+      - run:
+          name: $ npm install
+          command: npm install
+      - save_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
```

Comment:
This would work only if package.json had locked dependencies, we allow version intervals so this means that this can result in caching of stale dependencies. I'd suggest we remove caching for now.

Reply:
FWIW circle was doing this previously without us asking for it… I was trying to stick to as close to what we currently have.

Reply:
(removed)